tests/test_services_zeropush.py

```diff
@@ -117,7 +117,7 @@ def request_callback(request):
         send = zeropush.send_push_notification(devices, 'This is the message',
                                                sound="annoyingSound.mp3",
                                                badge_number=1,
-                                               payload={
+                                               custom={
```

Comment:
So is that a reason to not being compatible with older versions? Also docs should then be updated if this change needs to be there